setup.py

```diff
@@ -2,6 +2,7 @@
 # -*- coding: utf-8 -*-
 
 from kappa import __version__
+import codecs
```

Comment:
`io.open` is recommended for python >= 2.6.
I often use, `from io import open` to assure that python2 is using the python3 compatible `open()` which includes encoding specification.

https://mail.python.org/pipermail/python-list/2015-March/687124.html